mods/imperium/src/imperium-player.js

```diff
@@ -4751,6 +4769,7 @@ playerSelectInfantryToLand(sector) {
     if (id == "clear") {
```

Comment:
Try to avoid `==`. Should always be `===`. To avoid type issues.

Reply:
This is my bad. Will fix.

Should generally be doing this. There are still some older stragglers.

Reply:
eslint would complain or even fix this automatically